markdownlint/docs/TOP003.md

```diff
@@ -0,0 +1,138 @@
+# `TOP003` - Default Section Content
+
+Tags: `content`
+
+Aliases: `default-section-content`
+
+This rule is triggered when a section does not have the expected default content or structure. The affected sections include:
+
+- Lesson Overview
+- Assignment
+- Knowledge Check
+- Additional Resources
+
+Below are some examples of errors and their solutions:
+
+## Contains Content Before or After List
+
+There should be no content before or after the unordered list, other than the default content.
+
+Incorrect (content after the list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+
+Some additional content
+```
+
+Incorrect (content before the list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+Some additional content
+
+- LO item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+```
+
+## Contains Incorrect Default Content
+
+Each section should contain the correct default content.
+
+Incorrect (wrong default content before the list):
+
+```markdown
+### Lesson Overview
+
+This section has the incorrect text following the heading.
+
+- LO item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+```
+
+Incorrect (missing the list after the `Additional Resources` heading):
```

Comment:
Let's combine this and the below incorrect LO example, we don't really need to show both.